python/lsst/analysis/tools/actions/config/binning.py

```diff
@@ -0,0 +1,64 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+__all__ = ("MagnitudeBinnedMetricsConfig",)
+
+import lsst.pex.config as pexConfig
+
+
+class MagnitudeBinnedMetricsConfig(pexConfig.Config):
+    """Configuration for magnitude-binned metrics.
+
+    Bins may only be specified in mmag widths
+    """
+
+    mag_low_min = pexConfig.Field[int](
+        doc="Lower bound for the smallest magnitude bin in millimags",
+        default=15000,
```

Comment:
This being in millimags seems silly.

Reply:
Does going from 15 to 30 really make the maths than much harder than going from 15000 to 30000? It is much more understandable to go from 15 to 30 at a glance than 15000.

Reply:
I wanted to specify a min, max and width (rather than a number of bins with the width implicit) and not have rounding remove/add one bin because it ended on 23.9999999 instead of 24.0.